### PR TITLE
update docs around pyflyte run

### PR DESCRIPTION
**Pull request by samhita-alla**
# TL;DR
This PR updates docs around the pyflyte run command to indicate that it is no longer limited to single-script execution.

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
Fixes https://github.com/flyteorg/flyte/issues/3757

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

**Review by pingsutw — APPROVED** [review]

**Comment by pingsutw** [comment]
Thank you @samhita-alla
